src/tokens-studio/spectrum2-non-colors/spectrum2/layout/mobile.json

```diff
@@ -1429,7 +1429,7 @@
       }
     }
   },
-  "field-width": {
+  "field-width-small": {
```

Comment:
The script we run to do stuff like update UUIDs will also (for non-color token definitions) update the name field in the metadata to match the name here. It is probably a good practice to make this change explicitly, but it will be taken care of automatically if it gets missed anyway.

Reply:
thanks @mrcjhicks ! In that case, since it's a small fix, we can let the script update the name field.